Documentation/install/rhel/installing-workers.md

```diff
@@ -127,13 +133,14 @@ Note: These settings may not be all inclusive and will not represent relative no
 
 ### Set SELinux to Permissive mode
 
-It is required to run SELinux in Permissive mode. Running in Enforcing mode will block permissions for worker nodes. The following command will switch to Permissive mode until reboot.
+It is required to run SELinux in Permissive mode. Running in Enforcing mode will block permissions for worker nodes. Execute the following commands will enable permissive mode:
 
-```
-$ setenforce 0
-```
+1. Change the mode right for the current session: `setenforce 0`
```

Comment:
" mode right " sounds awkward. is it a standard phrase? or should it be simply "Change the permissions mode for the current session."